docs/Design/Design-Guide.md

```diff
@@ -1,151 +1,151 @@
-# Design guide

```

Comment:
I ran a command to normalize all files with the addition of the `.gitattributes` file. This file was the only one normalized.